estate_account/models/property.py

```diff
@@ -0,0 +1,33 @@
+from odoo import models, Command
+
+class Property(models.Model):
+    _inherit = "property"
+
+    def action_sold(self):
+        accepted_offer = self.offer_ids.get_accepted_offer()
+        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
+        self.env['account.move'].create({
+        'partner_id': accepted_offer.partner_id.id,
+        'move_type': 'out_invoice',
+        'journal_id': journal.id,
+        'invoice_line_ids': [
+            Command.create(
+                {"name": self.name,

```

Comment:
If the parent is multirecod you need to do the _inherit to support it too - otherwise it will crash on self.name

![image](https://github.com/AntonioMartinD/Odoo_Training/assets/35231827/e25c0167-c5df-44b3-b47e-f805cd29204e)